easybuild/easyconfigs/p/pygame/pygame-2.1.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'pygame'
+version = '2.1.0'
+
+homepage = 'https://www.pygame.org'
+description = """
+Pygame is a set of Python modules designed for writing video games. Pygame adds
+functionality on top of the excellent SDL library. This allows you to create
+fully featured games and multimedia programs in the python language.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SDL2_image', '2.6.3'),
+    ('SDL2_mixer', '2.6.3'),
+    ('SDL2_ttf', '2.20.2'),
+    ('PortMidi', '2.0.4'),
+    ('libpng', '1.6.37'),
+]
+
+use_pip = True
+
+_pygame_extra_base = ':'.join([
+    '$EBROOTSDL2_IMAGE',
+    '$EBROOTSDL2_MIXER',
+    '$EBROOTSDL2_TTF',
+    '$EBROOTLIBJPEGMINTURBO',
+    '$EBROOTLIBPNG',
+    '$EBROOTPORTMIDI',
+])
+
+exts_list = [
+    ('pygame', '2.1.0', {
+        'preinstallopts': 'PYGAME_EXTRA_BASE="%s"' % _pygame_extra_base,
```

Okay, that indeed seems to work, so we would need the following to make sure this works on Debian/Ubuntu (and maybe other distros):

```suggestion
        'preinstallopts': 'PORTMIDI_INC_PORTTIME=1 PYGAME_EXTRA_BASE="%s"' % _pygame_extra_base,
```